src/common/types/types.cpp

```diff
@@ -491,6 +492,7 @@ uint32_t Types::getDataTypeSize(const DataType& dataType) {
         for (auto& childType : structTypeInfo->getChildrenTypes()) {
             size += getDataTypeSize(*childType);
         }
+        size += NullBuffer::getNumBytesForNullValues(structTypeInfo->getChildrenNames().size());
```

Comment:
Now this function is becoming a bit confusing, can you add a comment to explain that this function is used for getting size of a value in the data type as in a tuple (i.e., ftable)?